linter/src/main/java/linter/App.java

```diff
@@ -3,12 +3,51 @@
  */
 package linter;
 
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+
 public class App {
-    public String getGreeting() {
-        return "Hello world.";
+    public static void main(String[] args) {
+        String PATH = "/Users/pousapan/codefellows-401/java-fundamentals!/linter/src/main/resources/gates.js";
+        System.out.println(readFile(PATH));
+
     }
 
-    public static void main(String[] args) {
-        System.out.println(new App().getGreeting());
+    public static String readFile(String path) {
+        int errorCount = 0;
+        String errorMessage = "";
+        try {
+            File file = new File(path);
+            int lineCount = 1;
+
+            Scanner sc = new Scanner(file);
+            while(sc.hasNextLine()) {
+                String line =sc.nextLine();
+                if ((line.endsWith("{")) || (line.endsWith("}"))){
```

Comment:
You could have condensed the `if` and `if else` conditionals into one. A common way to style a conditional with lots of conditions it to put each on it's own line:

```
if (line.endsWith("{") 
     || line.endsWith("}") 
     || line.contains("if") 
     || line.contains("else")
     || line.isEmpty()) {
```